src/components/Tabs/SettingsTab.tsx

```diff
@@ -280,74 +206,6 @@ export default function SettingsTab(): ReactElement {
             />
           </SettingsItem>
 
-          <SettingsItem
-            label={"Track path"}
```

Comment:
I also moved this section to its own checkbox collapse area, since there are lots of settings associated with the track path.

Reply:
<img width="476" height="198" alt="Image" src="https://github.com/user-attachments/assets/2b3a46d5-e706-4298-8b2c-027aae82412b" />